Podcast/EpisodeTableViewCell.swift

```diff
@@ -19,77 +19,38 @@ protocol EpisodeTableViewCellDelegate: class {
+    static let episodeTableViewCellHeight: CGFloat = 253
+    static let utilityButtonBarViewHeight: CGFloat = 48
     
     ///
     /// Mark: View Constants
     ///
-    var height: CGFloat = 253
     var seperatorHeight: CGFloat = 9
-    var episodeNameLabelY: CGFloat = 27
+    var episodeNameLabelY: CGFloat = 17
     var episodeNameLabelX: CGFloat = 86.5
     var episodeNameLabelRightX: CGFloat = 21
-    var episodeNameLabelHeight: CGFloat = 18
     var dateTimeLabelX: CGFloat = 86.5
-    var dateTimeLabelY: CGFloat = 47.5
-    var dateTimeLabelRightX: CGFloat = 18
-    var dateTimeLabelHeight: CGFloat = 30
     var descriptionLabelX: CGFloat = 17.5
-    var descriptionLabelY: CGFloat = 94
-    var descriptionLabelHeight: CGFloat = 54
-    var descriptionLabelRightX: CGFloat = 11.5
     var podcastImageX: CGFloat = 17.5
     var podcastImageY: CGFloat = 17
     var podcastImageSize: CGFloat = 60
-    
-    var bookmarkButtonHeight: CGFloat = EpisodeTableViewCell.bottomViewHeight
-    var bookmarkButtonWidth: CGFloat = 23
-    
-    var recommendedButtonWidth: CGFloat = 60
-    var recommendedButtonHeight: CGFloat = EpisodeTableViewCell.bottomViewHeight
-    var recommendedButtonRightX: CGFloat = 70
-
-    var buttonPadding: CGFloat = 10
-    let bottomViewInnerPadding: CGFloat = 18
-    
-    var lineSeperatorX: CGFloat = 18
-    var lineSeperatorHeight: CGFloat = 1
-    
-    var playButtonX: CGFloat = 18
-    var playButtonWidth: CGFloat = 75
-    var playButtonHeight: CGFloat = EpisodeTableViewCell.bottomViewHeight
-    
-    let moreButtonHeight: CGFloat = EpisodeTableViewCell.bottomViewHeight
-    let moreButtonWidth: CGFloat = 23
-    
-    var contextLabelX: CGFloat = 17
-    var contextLabelHeight: CGFloat = 30
-    var contextLabelRightX: CGFloat = 43
-    var contextImagesSize: CGFloat = 28
-    var feedControlButtonX: CGFloat = 345
-    var feedControlButtonHieght: CGFloat = 7.5
-    var feedControlButtonWidth: CGFloat = 13
-    var tagButtonsViewY: CGFloat = 160.5
-    var bottomViewHeight: CGFloat = EpisodeTableViewCell.bottomViewHeight
+    var tagButtonBottomMargin: CGFloat = 20
+    var utilityButtonBarViewHeight: CGFloat = EpisodeTableViewCell.utilityButtonBarViewHeight
     var mainViewHeight: CGFloat = 195
     
+    var marginSpacing: CGFloat = 6
+    var rightMarginSpacing: CGFloat = 11.5
+    
     ///
     /// Mark: Variables
     ///
     var episodeNameLabel: UILabel!
     var dateTimeLabel: UILabel!
     var descriptionLabel: UILabel!
-    var recommendedButton: RecommendButton!
-    var bookmarkButton: BookmarkButton!
     var seperator: UIView!
     var podcastImage: ImageView!
-    var lineSeperator: UIView!
-    var bottomLineSeperator: UIView!
-    var moreButton: MoreButton!
-    var playButton: PlayButton!
     var mainView: UIView! //main view
-    var bottomView: UIView! //bottom bar view with buttons
+    var utilityButtonBarView: EpisodeUtilityButtonBarView! //bottom bar view with buttons
```

Comment:
rather than utilityButtonBar why not bottomButtonBar, then we don't need this comment and it's clear how it's used throughout the code.

Reply:
I think I'll just keep it consistent with episodeUtilityButtonBarView